sopel/bot.py

```diff
@@ -60,17 +60,21 @@ def __init__(self, config, daemon=False):
             'medium': collections.defaultdict(list),
             'low': collections.defaultdict(list)
         }
+
         self.config = config
         """The :class:`sopel.config.Config` for the current Sopel instance."""
+
         self.doc = {}
         """
         A dictionary of command names to their docstring and example, if
         declared. The first item in a callable's commands list is used as the
         key in version *3.2* onward. Prior to *3.2*, the name of the function
         as declared in the source code was used.
         """
```

Comment:
This docstring could be rewritten to use Sphinx notations like `.. versionchanged:: 3.2`, with the notes below that. For example:

```py
        """A dictionary of command names to their documentation.

        Each command is mapped to its docstring and any available examples, if
        declared in the module's code.

        .. versionchanged:: 3.2
            Use the first item in each callable's commands list as the key,
            instead of the function name as declared in the source code.
        """
```